site/_config.yml

```diff
@@ -8,6 +8,7 @@ destination: local-generated
 twitter_url: https://twitter.com/asfbookkeeper
 
 versions:
+- "4.9.1"
```

Comment:
move this down to before `4.9.0`.

(I think the release script should be improved as well)

Reply:
Yes, I was thinking about this.
I had built the site locally and the result was looking fine to me.
Anyway I think you are right

I will change after @ivankelly commits his change on this same file for 4.8.2 release.